ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -1165,40 +1284,51 @@ tflite_close (const GstTensorFilterProperties *prop, void **private_data)
 static int
 tflite_loadModelFile (const GstTensorFilterProperties *prop, void **private_data)
 {
+  int ret = 0;
   TFLiteCore *core;
   tflite_option_s option = {};
 
   if (tflite_parseCustomOption (prop, &option) != 0) {
     g_printerr ("Failed to parse options to initialize tensorflow-lite model.");
-    return -1;
+    ret = -1;
+    goto done;
   }
 
   core = static_cast<TFLiteCore *> (*private_data);
 
   if (core != NULL) {
-    if (core->compareModelPath (option.model_file))
-      return 1; /* skipped */
+    if (core->compareModelPath (option.model_file)) {
+      ret = 1; /* skipped */
+      goto done;
+    }
 
     tflite_close (prop, private_data);
   }
 
   core = new TFLiteCore (prop);
   if (core == NULL) {
     g_printerr ("Failed to allocate memory for filter subplugin.");
-    return -1;
+    ret = -1;
+    goto done;
   }
 
   if (core->init (&option) != 0) {
     *private_data = NULL;
     delete core;
 
     g_printerr ("failed to initialize the object: Tensorflow-lite");
-    return -2;
+    ret = -2;
+    goto done;
   }
 
   *private_data = core;
 
-  return 0;
+done:
+  g_free ((gpointer) option.ext_delegate_path);
+  if (option.ext_delegate_kv_table)
+    g_hash_table_unref (option.ext_delegate_kv_table);
```

Comment:
Please clean them up by assigning null.
```
option.ext_delegate_path = nullptr;
option.ext_delegate_kvb_table = nullptr;
```

Reply:
Done in updated patchset.

Note that scope of option structure is local to this same tflite_loadModelFile() function.
So there was not risk for those pointers to be dereferenced later.